wiki/Overview.md

```diff
@@ -60,21 +60,23 @@ The following sections explain how to use these features.
 
 _flowR_ itself has two main ways to operate:
 
-- as a **server** which processes analysis and slicing requests (`--server` option)
-- as a **read-eval-print loop** (REPL) that can be accessed directly from the command line (default option)
+- as a [**server**](#the-server) which processes analysis and slicing requests (`--server` option)
+- as a [**read-eval-print loop** (REPL)](#the-read-eval-print-loop-repl) that can be accessed directly from the command line (default option)
 
 Besides these two ways, there is a [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=code-inspect.vscode-flowr) that allows you to use _flowR_ directly from within the editor (it is available on [open-vsx](https://open-vsx.org/extension/code-inspect/vscode-flowr) as well).
 Similarly, we offer an [Addin for RStudio](https://github.com/flowr-analysis/rstudio-addin-flowr), as well as an [R package](https://github.com/flowr-analysis/flowr-r-adapter).
 
 🐳️ If you use the docker-version, simply starting the docker container in interactive mode drops you right into the REPL (`docker run -it --rm eagleoutice/flowr:latest`), while launching with the `--server` argument starts the server (`docker run -it --rm eagleoutice/flowr:latest --server`).\
-⚒️ If you compile the _flowR_ sources yourself, you can access _flowR_ by first building the sources (`npm run build`) and executing then the root script (`node dist/src/flowr.js`).
+⚒️ If you compile the _flowR_ sources yourself, you can access _flowR_ by first building the sources (`npm run build`) and executing then the root script (`node dist/src/cli/flowr.js`).
```

Comment:
argh thanks, but this wouldn't have happened if we would generate the overview file as well so that would be a good backlog issue :D

Reply:
https://github.com/flowr-analysis/flowr/issues/2010 👍🏽